### DISCUSSION TITLE
Unable to install Web3uikit

### DISCUSSION THREAD

**Discussion by PsychoPunkSage** [web3ui/web3uikit#965] [Q&A] [answered]
Hi, I'm trying to add Web3uikit via yarn but it is constantly failing due to weird error
ERROR:
```
error An unexpected error occurred: "https://raw.githubusercontent.com/oscario2/wp5-cra-polyfill/91431fa2c67aaa1247309e04799254a2e8a949d7/package.json: ETIMEDOUT".
info If you think this is a bug, please open a bug report with the information provided in "/home/amaterasu/hh-fcc/nextjs-nft-marketplace-moralis/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command
```
please help in this regard

**Comment by AbhinavMV** [comment, marked as answer]
Hey @APrakash-913 try using the below command

`yarn add  @web3uikit/core @web3uikit/web3 @web3uikit/icons`

Please delete your `node_modules` folder before trying again.

Edit : if that does not work
run `yarn cache clean` and install dependencies again

**Reply by PsychoPunkSage** [reply]
Ya I have done that BUT still getting an ERROR⬇️
```
info There appears to be trouble with your network connection. Retrying...
error An unexpected error occurred: "https://raw.githubusercontent.com/oscario2/wp5-cra-polyfill/91431fa2c67aaa1247309e04799254a2e8a949d7/package.json: ETIMEDOUT".
info If you think this is a bug, please open a bug report with the information provided in "/home/amaterasu/hh-fcc/dryrun/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**Reply by AbhinavMV** [reply]
`There appears to be trouble with your network connection`

From the error it looks like a network problem and i am not able to replicate this issue either
Could you try with `npm install @web3uikit/core @web3uikit/web3 @web3uikit/icons` once ?

**Reply by vicentemunozlh** [reply]
I am having the same problem and `npm install @web3uikit/core @web3uikit/web3 @web3uikit/icons` also crashes .
```{}
npm WARN ERESOLVE overriding peer dependency
npm ERR! code ERR_SOCKET_TIMEOUT
npm ERR! network Socket timeout
npm ERR! network This is a problem related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network 
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vicente/.npm/_logs/2023-01-06T20_33_27_619Z-debug-0.log
```

**Reply by AbhinavMV** [reply]
Hey @vicentemunozlh 
Are you trying this with powershell? if so can you try installing packages in bash.

**Reply by vicentemunozlh** [reply]
Hello! I am using a bash terminal on VSC.

**Reply by AbhinavMV** [reply]
@vicentemunozlh  Could you try running `npm cache clean` 
and install dependencies again

**Reply by vicentemunozlh** [reply]
That worked ! thank you!
